js/navigation/Authentication/Authentication.js

```diff
@@ -0,0 +1,44 @@
+import React, { Component } from "react";
+import { AsyncStorage, View } from "react-native";
+import PropTypes from "prop-types";
+import { firebaseAuth } from "../../config/firebaseConfig";
+
+import { signIn } from "../../helpers/firebaseAuth";
+import styles from "./styles";
+import Loading from "../../components/Loading";
+
+export default class Authentication extends Component {
+  static propTypes = {
+    navigation: PropTypes.object.isRequired
+  };
+  constructor() {
```

Comment:
pass props in your constructor and super if you are using props in your components.